packages/manager/modules/pci/src/projects/project/streams/stream/delete/delete.html

```diff
@@ -0,0 +1,15 @@
+<form name="form" novalidate>
```

Comment:
```suggestion
<form novalidate>
```
seems like it's not used ?